src/shapes/SurfacePolyline.js

```diff
@@ -18,11 +18,13 @@
  */
 define([
         '../error/ArgumentError',
+        '../geom/Location',
```

Comment:
It seems Location isn't used in this class anymore. Please remove this dependency if possible.